src/main/webapp/js/components/table-page.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import ReactTable from 'react-table';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import 'react-table/react-table.css';
+
+import _ from 'lodash';
+import { fetchEntity, fetchFieldConfig } from '../actions/index';
+import getTableColumn from '../utils/table-utils';
+import { getVisibleFields } from '../selectors';
+
+class TablePage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loading: true,
+    };
+    this.props.fetchEntity(props.entityType);
+    this.props.fetchFieldConfig(props.entityType);
+  }
+
+  componentDidMount() {
+    this.setState({ loading: false });
+  }
+
+  render() {
+    const { entity, fieldConfigs } = this.props;
+    const entityList = Object.values(entity);
+    const fieldConfigList = Object.values(fieldConfigs);
+    const { loading } = this.state;
+    let tableColumns = [];
+    if (!_.isEmpty(fieldConfigs)) {
+      tableColumns = fieldConfigList
+        .sort((a, b) => a.fieldOrder - b.fieldOrder)
+        .map(config => getTableColumn(config));
+    }
+
+    return (
+      <ReactTable
+        data={entityList}
+        columns={tableColumns}
+        loading={loading}
+      />
+    );
+  }
+}
+
+const mapStateToProps = (state, props) => ({
+  entity: state.entity[props.entityType],
```

Comment:
you should add a new selector for this and map the object to array in this selector